sherpa/optmethods/tests/tstoptfct.hh

```diff
@@ -27,6 +27,8 @@
 // "Algorithm 566: Fortran Subroutines for Testing Unconstrained
 // Optimization Software.", ACM TOMS, VOL. 7, PAGES 14-41 AND 136-140, 1981
 //
+// See also https://www.osti.gov/biblio/6650344
+//
 // The comments on the solutions were taken from the site:
 //
 // http://www.uni-graz.at/imawww/kuntsevich/solvopt/results/moreset.html
```

Comment:
@dtnguyen2 This link does not exist any longer... Do you happen to have a copy? If not, shall we remove it?

Reply:
Oh, you're talkinga bout the uni-gratz URL. I thought for a second you were talking about the link I just added!

Reply:
@DougBurke Your link is fine and likely to be around for a while. It just shows link rot, in particular on personal websites.

So, I check the wayback machine and will add a new link.

Reply:
I've added the link @hamogu suggested but I'll leave this comment open for now